internal/writer/writer.go

```
@@ -0,0 +1,157 @@
+}
+
+func (w *CodeWriter) Bytes() []byte {
+	return []byte(w.builder.String())
+}
+
+func (w *CodeWriter) String() string {
+	return w.builder.String()
+}
+
+func (w *CodeWriter) WriteAll(items []string) {
+	slices.SortFunc(items, func(a, b string) int {
```

golangci-lint (2.12.2)

[error] unlambda: replace `func(a, b string) int {
	return cmp.Compare(a, b)
}` with `cmp.Compare`

(gocritic)